etc/docker/test/runtime.Dockerfile

```diff
@@ -0,0 +1,181 @@
+FROM almalinux:9.1 AS base
+    WORKDIR /usr/local/src
+    ARG PYTHON
+    ENV PYTHON=$PYTHON
+    ENV LANG=en_US.UTF-8
+    ENV LC_ALL=en_US.UTF-8
+    ENV CPLUS_INCLUDE_PATH="/usr/local/include/python${PYTHON}:/usr/include/python${PYTHON}"
+    ENV C_INCLUDE_PATH="/usr/include/python${PYTHON}"
+    ENV PYTHON_VENV="/opt/venv"
+    ENV PATH="${PYTHON_VENV}/bin:${PATH}"
+    ENV PYTHON_310_PATCH_VERSION="4"
+    ENV RUCIO_HOME="/opt/rucio"
+
+FROM base AS oracle-client
+    RUN dnf install -y libnsl libaio nodejs npm
+    RUN rpm -i https://download.oracle.com/otn_software/linux/instantclient/1912000/oracle-instantclient19.12-basiclite-19.12.0.0.0-1.x86_64.rpm && \
+        echo "/usr/lib/oracle/19.12/client64/lib" > /etc/ld.so.conf.d/oracle-instantclient.conf;
```

Comment:
No longer needed https://github.com/rucio/containers/pull/421 - @Geogouz pinging you just to confirm

Reply:
Checked with @Geogouz - let's leave it in for now